scripts/run_DESeq2.R

```diff
@@ -48,6 +48,7 @@ params <- set_up_platform_params(params)
 # Set this variable to be TRUE if you want to have separate plots of top genes as defined in the R-ODAF template
 params$R_ODAF_plots <- FALSE
 
+check_required_params()
```

Comment:
I guess when you call the function without params, it doesn't have them in memory when setup_functions.R gets sourced... might be able to fix with source(local = T)... this works for you locally though?

Reply:
whoops. That's a bug, should be passing in the params.